src/utils/mergeFile.js

```diff
@@ -2,26 +2,27 @@ import diff3Merge from 'diff3'
 
 const LINEBREAKS = /^.*(\r?\n|$)/gm
 
-export function mergeFile({
-  ourContent,
-  baseContent,
-  theirContent,
-  ourName = 'ours',
-  baseName = 'base',
-  theirName = 'theirs',
-  format = 'diff',
-  markerSize = 7,
-}) {
+export function mergeFile({ branches, contents }) {
+  const ourName = branches[1]
+  const theirName = branches[2]
+
+  const baseContent = contents[0]
+  const ourContent = contents[1]
+  const theirContent = contents[2]
+
   const ours = ourContent.match(LINEBREAKS)
   const base = baseContent.match(LINEBREAKS)
   const theirs = theirContent.match(LINEBREAKS)
 
   // Here we let the diff3 library do the heavy lifting.
   const result = diff3Merge(ours, base, theirs)
 
+  const markerSize = 7
```

Comment:
I would not get rid of this option and use:

```javascript
 mergeFile({ branches, contents, markerSize = 7 }) {

})
```

Just in case, so we don't remove something that may be needed.